sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -46,7 +46,7 @@ add the direct dependency to your project as follows.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-administration</artifactId>
-    <version>4.1.0-beta.5</version>
```

Comment:
This should remain beta - no?

Reply:
Same for other readme's too.

Reply:
I'm under the impression we want to keep our READMEs in main pointing to the latest stable version. I'll double-check just to make sure.